### ISSUE TITLE
Jaws doesn't read Preferences Overview

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ph-scale#351]
**Device**
Samsung
**OS**
Win 11
**Browser**
Chrome
**Problem Description**
For https://github.com/phetsims/qa/issues/1304
I could not get Jaws to read the overview tab in the preferences menu. It would read/interact with the other tabs just fine.

**Comment by marlitas** [comment]
This sounds like it might be a general issue with that tab right now in joist. I'll make an issue there. https://github.com/phetsims/joist/issues/1032
